README.md

```diff
@@ -271,6 +271,17 @@ Before starting these steps, perform [steps 1-5 from above](#setting-up-the-deve
   bundle exec rake assets:precompile
   ```
 
+ 1. Precompile the assets for Production Environment (else you may get squares instead of icons and other linking problems):
+
+  ```bash
+  RAILS_ENV=production rake assets:precompile
+  ```
+
+1. You may need to use the following cmd after the above cmd
+  ```bash
+  rm -rf public/assets
```

Comment:
I don't understand this. Why you think this is needed? You say that it _may_ be needed? When is it needed?

The purpose of the previous command `rake assets:precompile` is to generate precompiled files to `public/assets` directory, so removing that directory right after genereting the files doesn't make sense to me.